xml/update-alternatives.xml

```diff
@@ -0,0 +1,585 @@
+There are 2 choices for the alternative java (providing /usr/bin/java).
+
+  Selection    Path                                       Priority   Status
+------------------------------------------------------------
+* 0            /usr/lib64/jvm/jre-1.8.0-openjdk/bin/java   1805      auto mode
+  1            /usr/lib64/jvm/jre-1.7.0-openjdk/bin/java   1705      manual mode
+  2            /usr/lib64/jvm/jre-1.8.0-openjdk/bin/java   1805      manual mode
+
+Press &lt;enter> to keep the current choice[*], or type selection number:</screen>
+   <para>
+    Depending on your system and versions, the different Java version can
+    look a bit different.
+    After you have selected <literal>1</literal>, another run of <command>java</command>
+    prints the following result:
+   </para>
+   <screen>&prompt.user;<command>java</command> -version
+java version "1.7.0_141"
+OpenJDK Runtime Environment (IcedTea 2.6.10) (suse-42.3.1-x86_64)
+OpenJDK 64-Bit Server VM (build 24.141-b02, mixed mode)</screen>
+   <para>
+    Also keep in mind the following points:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      You are switching from auto mode to manual. This means, if you install
+      another Java version, the &as; will not touch the links nor it will
+      change the generic name.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      You are changing any slave links too which are associated to a master
+      link. In case of our Java example, this will change the many slave
+      links you can see from <xref linkend="sec.ua.displaying"/>. Check
+      with <command>update-alternatives</command> <option>--display java</option>.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    If you change your mind, you can always switch back.
+   </para>
+  </sect1>
+
+  <sect1 xml:id="sec.ua.install">
+   <title>Installing your own Alternatives</title>
+   <para>
+    To create some alternatives in your system, we assume the following
+    properties:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Two scripts, <command>foo-2</command> and <command>foo-3</command>,
+      which provides similar functionality.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      The scripts are stored in the <filename>/usr/local/bin</filename>
+      directory to avoid any conflicts with the system tools in
+      <filename>/usr/bin</filename>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      A master link <command>foo</command> which points either to
+      <command>foo-2</command> or <command>foo-3</command>.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    In case you want to provide alternatives in your system, follow these
+    steps: 
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Copy your scripts into the <filename>/usr/local/bin</filename> directory.
+     </para>
+    </step>
+    <step>
+     <para>
+      Make sure the scripts are executable:
```

Comment:
*Make sure the scripts are executable* -> Make the scripts executable

Reply:
done